plugins/reporters/spdx/src/main/kotlin/SpdxDocumentModelMapper.kt

```diff
@@ -172,7 +172,7 @@ internal object SpdxDocumentModelMapper {
                 comment = params.creationInfoComment,
                 created = Instant.now().truncatedTo(ChronoUnit.SECONDS),
                 creators = creators,
-                licenseListVersion = SpdxLicense.LICENSE_LIST_VERSION.substringBefore("-")
+                licenseListVersion = SpdxLicense.LICENSE_LIST_VERSION.split('.').take(2).joinToString(".")
```

Comment:
This solution works, but it may be more efficient this other implementation, saving the splitting and joining back
```suggestion
                val licenseList = SpdxLicense.LICENSE_LIST_VERSION
                licenseListVersion = licenseList.substring(0, licenseList.lastIndexOf("_"))
```

I do not really know much Kotlin, so take this with a pinch of salt.

Reply:
While `lastIndexOf("_")` would need to use `"."` instead, it would also not work for something like "1.2.3.4" as the input. Not that I'm expecting that to happen, but you never know 😉

Reply:
true!

Reply:
BTW, otherwise I would have simply used `substringBeforeLast('.')`.